march_gait_selection/src/march_gait_selection/dynamic_gaits/moveit_subgait.py

```diff
@@ -24,8 +24,9 @@ def __init__(self):
             rospy.logerr('Could not connect to move groups, aborting initialisation of the moveit subgait class')
             return
 
-        self._capture_point_msg = {'left_leg': None, 'right_leg': None}
         self._end_effectors = {'left_leg': 'left_foot', 'right_leg': 'right_foot'}
+        self._latest_msg_time = dict()
+        self._poses = dict()
```

Comment:
```suggestion
        self._latest_capture_point_msg_time = {'left_leg': None, 'right_leg': None}
        self._capture_point_position = {'left_leg': None, 'right_leg': None}
```
Or remove "latest_" in first name
Was there a reason you did not initialize these dict keys?